__test__/integration/debt_token_api/scenarios/orders.ts

```diff
@@ -41,4 +43,28 @@ export namespace Orders {
     };
 
     export const ALL_ORDERS = [ORDER_ONE, ORDER_TWO, ORDER_THREE];
+
+    export const CREDITOR_ONE = ACCOUNTS[5].address;
+    export const CREDITOR_TWO = ACCOUNTS[6].address;
+    export const APPROVEE = ACCOUNTS[7].address;
+
+    export const CREDITOR_ONE_ORDER: SimpleInterestLoanOrder = {
+        principalAmount: new BigNumber(15 * 10 ** 18),
+        principalTokenSymbol: ERC20TokenSymbol.ZRX,
+        interestRate: new BigNumber(4.135),
+        amortizationUnit: "months",
+        termLength: new BigNumber(14),
+        creditor: CREDITOR_ONE,
+        debtor: DEBTOR,
+    };
+
+    export const CREDITOR_TWO_ORDER: SimpleInterestLoanOrder = {
+        principalAmount: new BigNumber(3 * 10 ** 18),
+        principalTokenSymbol: ERC20TokenSymbol.REP,
+        interestRate: new BigNumber(4.325),
+        amortizationUnit: "months",
+        termLength: new BigNumber(5),
+        creditor: CREDITOR_TWO,
+        debtor: DEBTOR,
+    };
```

Comment:
Why create entirely new orders when we only want to vary the creditor?  Seems like it would make more sense to do something like:

```javascript
 export const CREDITOR_ONE_ORDER = {
    ...ORDER_ONE,
    creditor: CREDITOR_ONE,
}
```

Reply:
Agreed!